pkg/cri/sbserver/sandbox_run.go

```diff
@@ -154,15 +154,7 @@ func (c *criService) RunPodSandbox(ctx context.Context, r *runtime.RunPodSandbox
 			netnsMountDir = filepath.Join(c.config.StateDir, "netns")
 		}
 
-		uid := config.GetMetadata().GetUid()
```

Comment:
(I'm a newbie to containerd, so sorry for my stupid newbie question)
I'm quite unsure why you remove this code in the PR. You added 'uid := config.GetMetadata().GetUid() in first commit of the PR, but it is removed by here (2nd commit)...

Reply:
That was the result of a conversation in slack. The name includes the uid already